src/nodes/victron-virtual.js

```diff
@@ -546,8 +547,25 @@ module.exports = function (RED) {
               iface.Capacity = Number(config.battery_capacity)
             }
             if (config.default_values) {
+              let voltage = 24 // fallback
+
+              if (config.battery_voltage_preset === '12') {
```

Comment:
Can be made less repetitive:

```javascript
if (['12', '24', '48'].includes(config.battery_voltage_preset)) {
  voltage = Number(config.battery_voltage_preset)
} else if (config.battery_voltage_preset === 'custom') {
  ...
}
```

Maybe also extract the array into a variable:

```javascript
const BATTERY_DEFAULT_VOLTAGES = ['12', '24', '48']`;
```

This avoids hard-coding "magic numbers" in functions. We do not need this global variable anywhere else (yet), but it is cleaner to keep magic numbers out of functions, if possible.